src/prop/minisat/core/Solver.h

```diff
@@ -288,8 +288,14 @@ class Solver : protected EnvObj
                      bool b);  // Declare if a variable should be eligible for
                                // selection in the decision heuristic.
 
+ // Return the decision trail
  const vec<Lit>& getMiniSatDecisions() { return trail; }
 
+ // Return the order_heap, which is a priority queue of variables ordered with
+ // respect to the variable activity. The order heap is made available here
```

Comment:
@ajreynol Is this better for documenting getMiniSatOrderHeap? Should I put something more descriptive for getMiniSatDecisions, too?

Reply:
This looks fine to me.